framework/client/development.rst

```diff
@@ -23,6 +23,13 @@ Just install `Yarn`_ and execute the following commands:
   yarn setup
 
 
+Create your environment keys  in a ``/.env`` file by using ``yarn plop``, select ``Env``
```

Comment:
there is a shortcut in typing `yarn plop Env`

have you considered adding it to the yarn setup command? Will that overwrite an existing file? (this would be bad)